.circleci/config.yml

```diff
@@ -37,8 +37,6 @@ jobs:
     working_directory: ~/scalyr-agent-2
     docker:
       - image: circleci/python:3.8
-        environment:
-          PYTHONPATH=./scalyr_agent/third_party
```

Comment:
Those lines are not used since the tests and code rely on ``scalyr_init`` adding that directory to PYTHONPATH so it's better to remove it to reduce confusion.